src/OrchardCore/OrchardCore.DisplayManagement/Theming/ThemeSelectorResult.cs

```diff
@@ -13,7 +13,7 @@ public class ThemeSelectorResult
         /// <summary>
         /// Gets or sets the theme name.
         /// </summary>
-        /// <remarks>Avoid to use the theme name acciedently. <see cref="ThemeSelectorResult"/> is using the theme identifier instead.</remarks>
+        /// <remarks>Avoid to use the theme name accidentally. <see cref="ThemeSelectorResult"/> is using the theme identifier instead.</remarks>
```

Comment:
Gramatically this sentence makes no sense
```suggestion
        /// <remarks>This is the theme identifier. By default this is also the theme name, but if a custom identifier has been specified, please use the identifier.</remarks>
```

I would actually put it in the summary, noone will ever see it in remarks.

Reply:
I'll see the experience in VS & propbably VS Code, how the remarks are show up or they aren't appear at all

Reply:
> This is the theme identifier. By default this is also the theme name, but if a custom identifier has been specified, please use the identifier.

The theme name is not working at all?!!

Reply:
We could summarize it with what Jasmin suggested "Avoid using the theme name as a technical value." plus mentioning to use theme identifier instead

Reply:
This requires change when enabling theme from admin

as it uses theme name and not id